src/pages/JournalEntryFormPage/JournalEntryFormPage.tsx

```diff
@@ -34,10 +36,18 @@ const JournalEntryFormPage: React.FC = () => {
     const [probiotics, setProbiotics] = useState<boolean | null>(null);
     const [title, setTitle] = useState<string>('');
     const [description, setDescription] = useState<string>('');
+    const [error, setError] = useState<boolean>(false);
+    const [snackBarDetails, setSnackBarDetails] = useState<SnackBarDetails>({} as SnackBarDetails);
+
+    const dismissSnackBar = () => {
+        setSnackBarDetails({ ...snackBarDetails, show: false });
+    };
 
     const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
         e.preventDefault();
         console.log(moment(date).toISOString())
+        setSnackBarDetails({ error, show: true, message: `Journal entry ${entryUserId ? 'updated' : 'created'}!` })
+        // When connected to the backend, can use try catch and if its successful then navigate the user to their dashboard
```

Comment:
Will need to make sure the user is only navigated upon success, and also needs to make sure the Snackbar is displayed before taking the user to the dashboard. Maybe include a loader on the submit button till its settled, then show snackbar, then direct the user to their dashboard.